scripts/index.js

```diff
@@ -55,7 +55,13 @@ console.log(whoIsValyrian(characters));
 
 // What actor plays "Hot Pie" (and don't use IMDB)?
 
-
+function whoIsHotPie(chars) {
+    const him = chars.filter(function (each) {
+        return each.name === 'Hot Pie';
+    })
+    return him[0].playedBy;
```

Comment:
Cool. One tiny observation: it may or may not be worth it to add the extra step of extracting the values out of the `playedBy` array. (Your choice :) )